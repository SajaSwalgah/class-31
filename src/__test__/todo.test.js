import React from 'react';
import Enzyme, { mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Main from '../components/main.js';
import Header from '../components/header';
import Footer from '../components/footer';


Enzyme.configure({ adapter: new Adapter() });

describe('Our components are functioning', () => {
  it('Header renders', () => {
    let header = mount(<Header />);
    let h1 = header.find('h1');
    expect(h1.exists()).toBeTruthy();
  });

  it('Main renders', () => {
    let main = mount(<Main />);
    let span = main.find('span');
    expect(span.exists()).toBeTruthy();
  });

  it('Footer renders', () => {
    let foot = mount(<Footer />);
    let footer = foot.find('footer');
    expect(footer.exists()).toBeTruthy();
  });
});