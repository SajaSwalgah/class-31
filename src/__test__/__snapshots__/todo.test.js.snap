// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Our components are functioning render correctly to the DOM 1`] = `
<section>
  <form
    onSubmit={[Function]}
  >
    <input
      className="inputs"
      onChange={[Function]}
      placeholder="to do"
      type="text"
    />
    <br />
    <input
      className="inputs"
      onChange={[Function]}
      placeholder="assignee"
      type="text"
    />
    <br />
    <label>
      <input
        className="inputs"
        defaultValue="3"
        max="5"
        min="0"
        onChange={[Function]}
        placeholder="difficuly from 1-5"
        type="range"
      />
       
      <br />
    </label>
    <label
      className="container"
    >
      <input
        onClick={[Function]}
        type="checkbox"
      />
      <span
        id="state"
      >
         State
      </span>
      <span
        className="checkmark"
      />
    </label>
    <button>
      Add to List
    </button>
  </form>
  <p
    id="title"
  >
     To Do Items  
  </p>
</section>
`;
